crates/bevy_pbr/src/prepass/prepass.wgsl

```diff
@@ -0,0 +1,86 @@
+#import bevy_pbr::prepass_bindings
+#import bevy_pbr::mesh_functions
+
+struct Vertex {
+    @location(0) position: vec3<f32>,
+
+#ifdef VERTEX_UVS
+    @location(1) uv: vec2<f32>,
+#endif // VERTEX_UVS
```

Comment:
I was about to say it's because it's used for the pbr prepass, but that one uses the default Vertex struct from pbr_bindings, so I guess this indeed doesn't need to be here.

Reply:
It didn't need to be in the FragmentInput though